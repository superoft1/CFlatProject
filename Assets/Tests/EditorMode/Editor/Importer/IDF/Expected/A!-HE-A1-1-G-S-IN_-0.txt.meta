fileFormatVersion: 2
guid: 9fc58af0547d7564f8abc09357f45124
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
