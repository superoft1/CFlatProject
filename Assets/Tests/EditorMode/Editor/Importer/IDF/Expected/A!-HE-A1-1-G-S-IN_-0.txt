Pipe, , (1.0, 0.0, 0.0), (-373.0, 21.2, 5.5), False
(-373.1, 21.2, 5.5)
(-373.0, 21.2, 5.5)
FreeVertex: 1
WeldOlet, , (0.0, 0.0, 1.0), (-373.0, 21.2, 5.5), False
(-373.0, 21.2, 5.5)
(-373.0, 21.2, 5.5)
(-373.0, 21.2, 5.7)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-373.0, 21.2, 5.7), False
(-373.0, 21.2, 5.7)
(-373.0, 21.2, 5.8)
FreeVertex: 0
WeldNeckFlange, , (0.0, 0.0, 1.0), (-373.0, 21.2, 5.8), False
(-373.0, 21.2, 5.8)
(-373.0, 21.2, 5.8)
FreeVertex: 0
PressureGauge, PT_101_11_10_1070_1E, (0.0, 0.0, 1.0), (-373.0, 21.2, 5.8), False
(-373.0, 21.2, 5.9)
(-373.0, 21.2, 5.8)
FreeVertex: 1
Pipe, , (1.0, 0.0, 0.0), (-372.9, 21.2, 5.5), False
(-373.0, 21.2, 5.5)
(-372.8, 21.2, 5.5)
FreeVertex: 0
Elbow90, , (0.0, 0.0, 1.0), (-372.2, 21.2, 5.5), False
(-372.8, 21.2, 5.5)
(-372.2, 21.2, 4.9)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-372.2, 21.2, 4.8), False
(-372.2, 21.2, 4.9)
(-372.2, 21.2, 4.7)
FreeVertex: 0
WeldNeckFlange, , (0.0, 0.0, -1.0), (-372.2, 21.2, 4.7), False
(-372.2, 21.2, 4.6)
(-372.2, 21.2, 4.7)
FreeVertex: 1
FreeVertexTotal: 3
